packages/online/src/main.less

```diff
@@ -6,7 +6,6 @@
 @pf-padding: @grid-gutter-width / 2;
 
 #main {
```

Comment:
Out of curiosity, where is the `#main` selector declared in the recent versions?

Reply:
It's declared here: https://github.com/hawtio/hawtio-core/blob/4d91ea331156528df1342f12b935c4ca4e32de3d/src/app/page/page.component.ts#L89

Reply:
Is this really matching the `#main` selector? I would have expected it should use a CSS attribute selector, like `[role=main]`.

Reply:
Good point. I think recent versions don't set the "main" id anymore.

Reply:
OK that was my understanding. Thanks for the confirmation.